README.md

```diff
@@ -70,7 +73,7 @@ Contributions from the developer community are welcome. Please refer to the cont
 
 # Example
```

Comment:
As a note: when the Rust client is ready it would be nice to put it here in the example section instead of the Test client 😃